components/02-components/tooltips/README.md

```diff
@@ -1 +1,12 @@
-*There are no notes for this component yet.* 
+Tooltips contain contextual annotations or helper text on demand when a user hovers or focuses on an element, usually in informational icon.
+
+## When to use
+- When a setting or control is highly specific to the FEC’s unique subject matter or context </br>
+_Ex: The “[Restrict contributions](https://www.fec.gov/data/receipts/?data_type=processed&two_year_transaction_period=2018&min_date=01%2F01%2F2017&max_date=03%2F02%2F2018)” filter_
```

Comment:
For some reason the single underscore italic isn't rendering on the notes section.